cmd/root_test.go

```diff
@@ -36,7 +36,19 @@ func (mock mockLaunch) Kill(os.Signal) {}
 func (mock mockLaunch) Clean() {}
 
 func setup() {
-	configNew = func(confPath string) (config.Config, error) { return config.Config{}, nil }
+	configNew = func(confPath string) (config.Config, error) {
+		return config.Config{
```

Comment:
The config.Config have deep nesting.  I think it is better to declare config.Config  at return above.

Reply:
I made an effort to make this nest shallow.